template/files/Gemfile.tt

```diff
@@ -5,6 +5,7 @@ gem 'tzinfo-data', platforms: [:mingw, :mswin, :jruby]
 
 gem 'puma', require: false
 
+gem 'redis-rails'
```

Comment:
Why did you add `redis-rails`?
Is it required by something else?

Reply:
added it as it wa sin banking-adapter, removed